lecture_notes/Exercises/lesson_1_finished.md

```diff
@@ -1,5 +1,5 @@
 
-# Exercises - finished
+# Exercises - Finished
 ## Retirement
 * Write a program that receives your age and tells you **how many years** you have left **before you retire**. (The age of retirement should be **65**.
 You will only need `input()`, `print()`, `int()`, `str()` and `float()` for casting
```

Comment:
`input()` and `print()` for interaction with the user and  `int()`, `str()` and `float()` for casting (getting numbers from strings)

Reply:
Aby to měli jasnější. K čemu btw potřebujou float?